.github/workflows/release.yml

```diff
@@ -23,10 +23,20 @@ jobs:
           php-version: '8.1'
           coverage: none
       -
+        name: Prepare
         run: |
           git config --global user.email "devel@keboola.com"                           
           git config --global user.name "Keboola CI"
           LATEST_TAG=$(git -c 'versionsort.suffix=-' ls-remote --exit-code --refs --sort='version:refname' --tags origin '*.*.*' | cut --fields=2 | sed -E --expression="s/(refs\/tags\/)v/\1/g" | sort | tail --lines=1)
           git fetch origin $LATEST_TAG:$LATEST_TAG
           composer install
-          php vendor/bin/monorepo-builder release ${{ inputs.version }}
+
+      -
+        name: Release new version
+        run: php vendor/bin/monorepo-builder release ${{ inputs.version }}
+
+      - name: Push changes
```

Comment:
Niesom si teda este uplne isty ci to bude fungovat ale nemam to asi jak vyskusat, commitnute by to malo byt, potrebujem akurat pushnut ten commit kde su tie composer zavyslosti. Tag by v tomto momete uz mal byt pushnuty preto to robi `PushTagReleaseWorker`